frontend/src/utils/address.ts

```diff
@@ -147,6 +147,12 @@ const jsonFetch = (url: string) => fetch(url).then((res) => res.json());
 
 // Returns true if the address owns any POAP tokens
 const hasPOAPs = async (address: string) => {
-  const poaps = await jsonFetch(`https://api.poap.xyz/actions/scan/${address}`);
-  return poaps.length > 0 ? true : false;
+  try {
+    const poaps = await jsonFetch(`https://api.poap.xyz/actions/scan/${address}`);
+    return poaps.length > 0 ? true : false;
+  } catch (err) {
+    console.warn('Error in hasPOAPs');
+    console.warn(err);
+    return false;
```

Comment:
Hmm are we ok with assuming `false` when this query fails? Maybe we should default to `true` to be more conservative?

Reply:
I don't think so. A false positive could cause a lot of confusion for the user, especially if they're using an address that is totally safe.

Reply:
Yea, the third options is return something other tan true/false and say "may have POAPs, not sure, the check failed" and let the user handle that information, but that feels overly complex

Reply:
We're doing this "to be helpful" to users—this list is not comprehensive and there's other things that could break privacy, so I think false is ok